code/modules/surgery/organs/tongue.dm

```diff
@@ -310,3 +310,45 @@
 /obj/item/organ/tongue/ethereal/Initialize(mapload)
 	. = ..()
 	languages_possible = languages_possible_ethereal
+
+//Sign Language Tongue - yep, that's how you speak sign language.
+/obj/item/organ/tongue/tied
+	name = "tied tongue"
+	desc = "Cat's got your tounge?"
+	say_mod = "signs"
+	icon_state = "tonguetied"
+	modifies_speech = TRUE
+
+/obj/item/organ/tongue/tied/Insert(mob/living/carbon/M)
+	. = ..()
+	M.verb_ask = "signs"
+	M.verb_exclaim = "signs"
+	M.verb_whisper = "subtly signs"
+	M.verb_sing = "rythmically signs"
+	M.verb_yell = "emphatically signs"
+	ADD_TRAIT(M, TRAIT_SIGN_LANG, "tongue")
+	REMOVE_TRAIT(M, TRAIT_MUTE, "tongue")
+
+//Here lies tied/Remove, you will be missed for your simplicity and aheal-breaking properties.
+
+//Thank you Jwapplephobia for helping me with this literal hellcode
+
+/obj/item/organ/tongue/tied/handle_speech(datum/source, list/speech_args)
+	var/new_message
+	var/message = speech_args[SPEECH_MESSAGE]
+	var/exclamation_used = FALSE
+	var/question_mark_used = FALSE
+	var/mob/living/carbon/M = owner
+	for(var/i in 1 to length(message))
+		if(findtext("!", message[i]))
+			exclamation_used = TRUE
+		else if (findtext("?", message[i]))
```

Comment:
We might also want to prevent homoglyphs too.  Such as `ʔ`

Reply:
What homoglyphs do people regularly use in a normal game? I can't really think of any.

Reply:
Nevermind I think I finally understand what you're saying
If you mean to remove things that LOOK like question marks & exclamation points, I might be able to do that, although I doubt that players would need to resort to using homoglyphs for their speech, due to the visualization with their eyebrows.